locustfile.py

```diff
@@ -18,273 +18,398 @@
 # Avoid "Too many open files" error
 resource.setrlimit(resource.RLIMIT_NOFILE, (9999, 999999))
 
-API_KEY = os.environ['FEC_API_KEY']
+API_KEY = os.environ["FEC_API_KEY"]
 
 # it seems like AUTH is NOT used for current tests
 try:
-    AUTH = (os.environ['FEC_USERNAME'], os.environ['FEC_PASSWORD'])
+    AUTH = (os.environ["FEC_USERNAME"], os.environ["FEC_PASSWORD"])
 except KeyError:
     AUTH = None
 
 CYCLES = range(1980, 2018, 2)
 CANDIDATES = [
-    'bush'
-    'cheney',
-    'gore',
-    'lieberman',
-    'kerry',
-    'edwards',
-    'obama',
-    'biden',
-    'mccain',
-    'palin',
-    'clinton',
-    'sanders'
-    'omalley',
-    'rubio',
-    'graham',
-    'kasich',
-]
-TERMS = [
-    'embezzle',
-    'email',
-    'department',
-    'contribution',
-    'commission'
+    "bush" "cheney",
```

Comment:
I think this and line 41 were supposed to have commas - could you add them please?